app/views/components/_published-dates.html.erb

```diff
@@ -1,13 +1,22 @@
 <%
   published ||= false
-  history ||= []
-  history = Array(history)
+  data ||= {}
   last_updated ||= false
   link_to_history ||= false
-  history_class = "app-c-published-dates--history" if history.any?
+  notification_button_visible ||= false
+
+  history ||= []
+  history = Array(history)
+  id = "history" if history.any?
+  data[:module] = "gem-toggle" if history.any?
```

Comment:
I wonder if writing this long hand, although less tidy, would be fractionally more efficient?

```
if history.any?
  id = "history"
  data[:module] = "gem-toggle"
end
```

Non blocking, but something for someone with more programming knowledge than me to answer.